v1/wd.c

```diff
@@ -258,8 +258,11 @@ static int get_int_attr_all(struct dev_info *dinfo)
 	dinfo->node_id = get_int_attr(dinfo, "node_id");
 
 	ret = get_int_attr(dinfo, "flags");
-	if (ret < 0 || (unsigned int)ret & WD_UACCE_DEV_SVA)
+	if (ret < 0)
 		return ret;
+	else if ((unsigned int)ret & WD_UACCE_DEV_SVA)
+		return -ENODEV;
+
```

Comment:
According to your comment, checking flag is unnecessary since there's no flag on return value. Then why do you still check it? Why not delete it at all?

Reply:
If the device is  SVA，we should skip it by ENODEV.
If we not change it, it will return an number bigger than 0(like 129 read from uacce flags), the last caller wd_get_available_dev_num may take it as a normal device number.

Reply:
PS，wd_get_available_dev_num is a no-sva api，so we need skip sva mode device。